src/config/api.js

```diff
@@ -6,6 +6,5 @@ export const SingleCoin = (id) =>
 
```

Comment:
I'm curious if, rather than this file just returning the API urls, if it could instead actually run the API calls and return the resulting data? This may not make sense to do now though if you plan to later manage this data/its interactions inside a Redux slice. 🤷‍♀️